src/cmd/run.rs

```diff
@@ -136,6 +136,7 @@ struct ScanGitVCSReference {
 }
 
 impl ScanGitVCSReference {
+    /// Instantiate ScanGitVCSReference instance
```

Comment:
[nit] In constructors, we usually just say "Instantiate an instance" or something, no need to copy the name of the type- it's already right there (since a constructor for `T` is called as `T::new()`).